python/pyspark/context.pyi

```diff
@@ -0,0 +1,179 @@
+#
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+# Stubs for pyspark.context (Python 3.5)
```

Comment:
Why are some of them generated using Python 3.5 and some 3.7?
```
# Stubs for pyspark.context (Python 3.5)
```
I'd say that we don't want to use 3.5 since we've dropped this anyway.

Reply:
These can be safely removed (have been automatically added by stubgen  and in practice don't really reflect much more beyond the interpreter used at the moment).